app/controllers/feedbacks_controller.rb

```diff
@@ -5,8 +5,8 @@ class FeedbacksController < ApplicationController
 
 
  get '/feedbacks/new' do
-   if session[:user_type]="student"
-     @student = Student.find(session[:user_id])
+   if authenticate_student!
```

Comment:
This should just be calling that method and handling redirection in that private `authenticate_student!` method